### PR TITLE
fix(cdsctl,api): return msg error wf import

### LINKED ISSUE #3806: fix(api): import workflow with invalid pipeline
return {"id":12,"message":"pipeline does not exist","uuid":"09eb0130-0f64-11e9-9812-0242ac110004"}

should return the name of invalid pipeline

cdsctl amin errors get 09eb0130-0f64-11e9-9812-0242ac110004
Message: pprofLabel>recoverWrap>Handle>postWorkflowImportHandler>ParseAndImport>IsValid>checkPipeline: pipeline does not exist (caused by: Unable to import workflow ...

### PR DESCRIPTION
**Pull request by yesnault**
close #3806

Signed-off-by: Yvonnick Esnault <yvonnick.esnault@corp.ovh.com>

### CODE REVIEW

File: cli/cdsctl/application.go [hunk 1]
```diff
@@ -126,19 +126,13 @@ func applicationImportRun(c cli.Values) error {
 	formatStr, _ := exportentities.GetFormatStr(format)
 
 	msgs, err := client.ApplicationImport(c.GetString(_ProjectKey), contentFile, formatStr, c.GetBool("force"))
+	for _, msg := range msgs {
+		fmt.Println(msg)
+	}
 	if err != nil {
```
Comment (fsamin) [inline cli/cdsctl/application.go]: just `return err`

File: cli/cdsctl/environment.go [hunk 2]
```diff
@@ -114,13 +114,12 @@ func environmentImportRun(c cli.Values) error {
...
-	}
-
 	for _, s := range msgs {
 		fmt.Println(s)
 	}
+	if err != nil {
+		return err
+	}
 
 	return nil
```
Comment (fsamin) [inline cli/cdsctl/environment.go]: just `return err`

File: cli/cdsctl/workflow_import.go [hunk 3]
```diff
@@ -42,13 +42,12 @@ func workflowImportRun(c cli.Values) error {
...
-	}
-
 	for _, s := range msgs {
 		fmt.Println(s)
 	}
+	if err != nil {
+		return err
+	}
 
 	return nil
```
Comment (fsamin) [inline cli/cdsctl/workflow_import.go]: just `return err`

File: engine/api/environment_import.go [hunk 4]
```diff
@@ -73,7 +73,6 @@ func (api *API) postEnvironmentImportHandler() service.Handler {
 			if sdk.ErrorIsUnknown(globalError) {
 				return globalError
 			}
-			log.Warning("%v", globalError)
```
Comment (fsamin) [inline engine/api/environment_import.go]: if you want to remove it, you can remove it from:
https://github.com/ovh/cds/blob/7ea89dd5a82080f5e80716df1a3253cb81d1ea30/engine/api/pipeline_import.go#L99

**Comment by ovh-cds** [comment]
CDS Report ut-engine#11206.0 ✘  
* Stage 1
  * Engine Test with DB ✘

File: engine/api/environment_import.go (hunk 4 above)
Comment (yesnault) [reply engine/api/environment_import.go]: fixed

**Pull request merged**